src/tests/ftest/util/dmg_utils.py

```diff
@@ -622,6 +625,7 @@ def pool_create(self, scm_size, uid=None, gid=None, nvme_size=None,
         data["ranks"] = ",".join([str(r) for r in output["response"]["tgt_ranks"]])
         data["scm_per_rank"] = output["response"]["tier_bytes"][0]
         data["nvme_per_rank"] = output["response"]["tier_bytes"][1]
+        data["memfile_per_rank"] = output["response"]["mem_file_bytes"]
```

Comment:
This doesn't address DAOS-16709 - that requires not running `json.loads("")`.  It seems that something like this will be needed as part of [adding new phase 2 tests](https://daosio.atlassian.net/browse/DAOS-16680), so maybe it should be handled then.

Reply:
Right, thanks for diagnosing and taking this one on. Closing... Maybe you could fix the comment above describing the JSON output for pool create in your fix for this please.